Source/HyprMXAdapter.swift

```diff
@@ -191,3 +206,18 @@ extension HyprMXAdapter: HyprMXInitializationDelegate {
         initializationCompletion = nil
     }
 }
+
+internal extension HyprConsentStatus {
+    var description: String {
+        switch self.rawValue {
```

Comment:
Do we need to switch on the `rawValue`, or can we just switch on the enum itself?

Reply:
Hmm, I guess [that does work](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-hyprmx/pull/13/commits/b4b41b31f5a0d9149dbb27b0d3793b3af498b1ec).
I remember writing that part and starting to type in "`case CONSENT_...`", then backtracking because the enum names weren't appearing in autocomplete so I assumed there was some reason I couldn't access them there.
So that's a lesson about over-reliance on autocomplete.